### DISCUSSION TITLE
ceilling detection plans

### DISCUSSION THREAD

**Discussion by diego95iot** [google/model-viewer#2129] [Ideas]
I want to know if it is possible to place objects by planting them on the ceiling, i know that it is possible in AR WEB with arKit and in arCore.

If this is not possible, is it being worked on to introduce in future releases?

Thank you.

**Comment by elalish** [comment]
not yet, though I suppose it should be possible. I'm a little concerned mostly because ceilings often lack texture, which will make them difficult for AR to track properly.

**Reply by Pruebas11111** [reply]
@elalish Hello. And, actually, does this work?  Place the objects on the ceiling? And if it does work, any information on how to do it? I know how to place it on the floor and wall. But not on the ceiling. Thank you!!!

**Reply by elalish** [reply]
Not currently - PRs welcome!
